goerli/373450.json

```diff
@@ -0,0 +1 @@
+[{"message":{"validator_index":"373450","from_bls_pubkey":"0x90fa22da54d08a8b0e094f3dd162323c0ecc5edae4901e38844c63bb6b1e9028a086027c22dce3372516e80513b942e7","to_execution_address":"0x5385eefdae761edc311f08474a6ca0bf2eb8f2a7"},"signature":"0xa5d0246db4bf20ce5f31717dd3b3c2960505d97ed28e9b704dc1f25faf71a48239bd6dcb6f33aec77d77af7763295e9d07f40d319aafc2ec9c05c6e3dddc7642380deb1d2a9a9c319e3a01e1c50466992e77b7e2b6b6995ed5f74d4a9605b2fa"},{"message":{"validator_index":"373463","from_bls_pubkey":"0x8d924ec8b4cf4752f5ad56337dd1205ad4b05f647b0ffa98a5921602ff8bd60d238b15796f4e42c8021931a39ebb6f9e","to_execution_address":"0x34df269e039bda53550ceaf4bedcb118cde71b3d"},"signature":"0xb8f95dcf2e3efd7f024570997ca2ce60aeb28414d1bb64a60923128ececc6c43581e097c06fed45484a9d37c532412ae15e7d12f978c232990dd7880db2f7f3afb0b4f9192894710bfd551457f80a8a4cb7af46634be266129e6daaa56286e2a"}]
```

Comment:
Validator file goerli/373450.json must only contain a message for validator 373450 - it may not contain 373463 in it. The signature for 373450 is also incorrect. However, the signature for 373463 is correct - but cannot be listed in this file together.